src/main/java/edu/tseidler/model/Board.java

```diff
@@ -4,9 +4,9 @@
 
 public class Board {
     private static final int MINIMUM_SIZE_AND_WINNING_NUMBER = 3;
-    public final int maxRow;
-    public final int maxCol;
-    public final int winningNumber;
+    private final int maxRow;
```

Comment:
I would suggest to extract these variables to a new class, something like BoardParameters. Then you would have only one object to pass to a Drawer, and in Drawer class you would have also just one object, such that code will be more DRY.
It will allow you to change these three getters into just one, and you will not repeat code structure like you do now, because you have:

```
-    public final int maxRow;
-    public final int maxCol;
-    public final int winningNumber;
```

in Board class and in Drawer class, and that's bad.